output/cloud/expv2/pbcloud/metric.proto

```diff
@@ -140,13 +140,14 @@ message TrendHdrValue {
   double min_value = 7;
   double max_value = 8;
 
+  // histogram parameter - value multiplier aka smallest value
```

Comment:
I don't think that it's critical since, in any case, the number is essential, but what's the reason for moving it?

Reply:
Good catch, I forgot the order